Connecting to database specified by database.yml
Unable to load applicant, underlying cause No such file to load -- applicant 

 /Users/Arnav/.rvm/gems/ruby-1.9.3-p194/gems/activesupport-3.2.8/lib/active_support/dependencies.rb:317:in `rescue in depend_on'
/Users/Arnav/.rvm/gems/ruby-1.9.3-p194/gems/activesupport-3.2.8/lib/active_support/dependencies.rb:312:in `depend_on'
/Users/Arnav/.rvm/gems/ruby-1.9.3-p194/gems/activesupport-3.2.8/lib/active_support/dependencies.rb:225:in `require_dependency'
/Users/Arnav/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:767:in `try_to_load_dependency'
/Users/Arnav/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:782:in `block in require_fixture_classes'
/Users/Arnav/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:779:in `each'
/Users/Arnav/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:779:in `require_fixture_classes'
/Users/Arnav/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:762:in `fixtures'
/Users/Arnav/CampusHunter/test/test_helper.rb:10:in `<class:TestCase>'
/Users/Arnav/CampusHunter/test/test_helper.rb:5:in `<top (required)>'
/Users/Arnav/CampusHunter/test/unit/applicants_test.rb:1:in `require'
/Users/Arnav/CampusHunter/test/unit/applicants_test.rb:1:in `<top (required)>'
/Users/Arnav/.rvm/gems/ruby-1.9.3-p194/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:10:in `require'
/Users/Arnav/.rvm/gems/ruby-1.9.3-p194/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:10:in `block (2 levels) in <main>'
/Users/Arnav/.rvm/gems/ruby-1.9.3-p194/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:9:in `each'
/Users/Arnav/.rvm/gems/ruby-1.9.3-p194/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:9:in `block in <main>'
/Users/Arnav/.rvm/gems/ruby-1.9.3-p194/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:4:in `select'
/Users/Arnav/.rvm/gems/ruby-1.9.3-p194/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:4:in `<main>'
Connecting to database specified by database.yml
Unable to load applicant, underlying cause No such file to load -- applicant 

 /Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activesupport-3.2.8/lib/active_support/dependencies.rb:317:in `rescue in depend_on'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activesupport-3.2.8/lib/active_support/dependencies.rb:312:in `depend_on'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activesupport-3.2.8/lib/active_support/dependencies.rb:225:in `require_dependency'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:767:in `try_to_load_dependency'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:782:in `block in require_fixture_classes'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:779:in `each'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:779:in `require_fixture_classes'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:762:in `fixtures'
/Users/Shilpa/CampusHunter/test/test_helper.rb:10:in `<class:TestCase>'
/Users/Shilpa/CampusHunter/test/test_helper.rb:5:in `<top (required)>'
/Users/Shilpa/CampusHunter/test/unit/applicants_test.rb:1:in `require'
/Users/Shilpa/CampusHunter/test/unit/applicants_test.rb:1:in `<top (required)>'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:10:in `require'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:10:in `block (2 levels) in <main>'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:9:in `each'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:9:in `block in <main>'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:4:in `select'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:4:in `<main>'
Connecting to database specified by database.yml
Unable to load applicant, underlying cause No such file to load -- applicant 

 /Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activesupport-3.2.8/lib/active_support/dependencies.rb:317:in `rescue in depend_on'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activesupport-3.2.8/lib/active_support/dependencies.rb:312:in `depend_on'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activesupport-3.2.8/lib/active_support/dependencies.rb:225:in `require_dependency'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:767:in `try_to_load_dependency'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:782:in `block in require_fixture_classes'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:779:in `each'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:779:in `require_fixture_classes'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:762:in `fixtures'
/Users/Shilpa/CampusHunter/test/test_helper.rb:10:in `<class:TestCase>'
/Users/Shilpa/CampusHunter/test/test_helper.rb:5:in `<top (required)>'
/Users/Shilpa/CampusHunter/test/functional/applicant_controller_test.rb:1:in `require'
/Users/Shilpa/CampusHunter/test/functional/applicant_controller_test.rb:1:in `<top (required)>'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:10:in `require'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:10:in `block (2 levels) in <main>'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:9:in `each'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:9:in `block in <main>'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:4:in `select'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:4:in `<main>'
  [1m[36m (0.3ms)[0m  [1mbegin transaction[0m
  [1m[35mFixture Delete (0.9ms)[0m  DELETE FROM "applicants"
  [1m[36mFixture Insert (0.2ms)[0m  [1mINSERT INTO "applicants" ("Name", "Score", "Branch", "Role", "CodePairing", "PairingStatus", "FirstTech", "FirstStatus", "SecondTech", "Result", "Comment") VALUES ('MyString', 0, 'MyString', 'MyString', 'MyString', 'MyString', 'MyString', 'MyString', 'MyString', 'MyString', 'MyString')[0m
SQLite3::ConstraintException: applicants.created_at may not be NULL: INSERT INTO "applicants" ("Name", "Score", "Branch", "Role", "CodePairing", "PairingStatus", "FirstTech", "FirstStatus", "SecondTech", "Result", "Comment") VALUES ('MyString', 0, 'MyString', 'MyString', 'MyString', 'MyString', 'MyString', 'MyString', 'MyString', 'MyString', 'MyString')
  [1m[35m (1.5ms)[0m  rollback transaction
Connecting to database specified by database.yml
Unable to load applicant, underlying cause No such file to load -- applicant 

 /Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activesupport-3.2.8/lib/active_support/dependencies.rb:317:in `rescue in depend_on'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activesupport-3.2.8/lib/active_support/dependencies.rb:312:in `depend_on'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activesupport-3.2.8/lib/active_support/dependencies.rb:225:in `require_dependency'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:767:in `try_to_load_dependency'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:782:in `block in require_fixture_classes'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:779:in `each'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:779:in `require_fixture_classes'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:762:in `fixtures'
/Users/Shilpa/CampusHunter/test/test_helper.rb:10:in `<class:TestCase>'
/Users/Shilpa/CampusHunter/test/test_helper.rb:5:in `<top (required)>'
/Users/Shilpa/CampusHunter/test/unit/applicants_test.rb:1:in `require'
/Users/Shilpa/CampusHunter/test/unit/applicants_test.rb:1:in `<top (required)>'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:10:in `require'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:10:in `block (2 levels) in <main>'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:9:in `each'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:9:in `block in <main>'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:4:in `select'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:4:in `<main>'
Connecting to database specified by database.yml
Unable to load applicant, underlying cause No such file to load -- applicant 

 /Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activesupport-3.2.8/lib/active_support/dependencies.rb:317:in `rescue in depend_on'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activesupport-3.2.8/lib/active_support/dependencies.rb:312:in `depend_on'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activesupport-3.2.8/lib/active_support/dependencies.rb:225:in `require_dependency'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:767:in `try_to_load_dependency'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:782:in `block in require_fixture_classes'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:779:in `each'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:779:in `require_fixture_classes'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:762:in `fixtures'
/Users/Shilpa/CampusHunter/test/test_helper.rb:10:in `<class:TestCase>'
/Users/Shilpa/CampusHunter/test/test_helper.rb:5:in `<top (required)>'
/Users/Shilpa/CampusHunter/test/functional/applicant_controller_test.rb:1:in `require'
/Users/Shilpa/CampusHunter/test/functional/applicant_controller_test.rb:1:in `<top (required)>'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:10:in `require'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:10:in `block (2 levels) in <main>'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:9:in `each'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:9:in `block in <main>'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:4:in `select'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:4:in `<main>'
  [1m[36m (0.3ms)[0m  [1mbegin transaction[0m
  [1m[35mFixture Delete (0.9ms)[0m  DELETE FROM "applicants"
  [1m[36mFixture Insert (0.2ms)[0m  [1mINSERT INTO "applicants" ("Name", "Score", "Branch", "Role", "CodePairing", "PairingStatus", "FirstTech", "FirstStatus", "SecondTech", "Result", "Comment") VALUES ('MyString', 0, 'MyString', 'MyString', 'MyString', 'MyString', 'MyString', 'MyString', 'MyString', 'MyString', 'MyString')[0m
SQLite3::ConstraintException: applicants.created_at may not be NULL: INSERT INTO "applicants" ("Name", "Score", "Branch", "Role", "CodePairing", "PairingStatus", "FirstTech", "FirstStatus", "SecondTech", "Result", "Comment") VALUES ('MyString', 0, 'MyString', 'MyString', 'MyString', 'MyString', 'MyString', 'MyString', 'MyString', 'MyString', 'MyString')
  [1m[35m (2.3ms)[0m  rollback transaction
Connecting to database specified by database.yml
Unable to load applicant, underlying cause No such file to load -- applicant 

 /Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activesupport-3.2.8/lib/active_support/dependencies.rb:317:in `rescue in depend_on'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activesupport-3.2.8/lib/active_support/dependencies.rb:312:in `depend_on'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activesupport-3.2.8/lib/active_support/dependencies.rb:225:in `require_dependency'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:767:in `try_to_load_dependency'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:782:in `block in require_fixture_classes'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:779:in `each'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:779:in `require_fixture_classes'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194/gems/activerecord-3.2.8/lib/active_record/fixtures.rb:762:in `fixtures'
/Users/Shilpa/CampusHunter/test/test_helper.rb:10:in `<class:TestCase>'
/Users/Shilpa/CampusHunter/test/test_helper.rb:5:in `<top (required)>'
/Users/Shilpa/CampusHunter/test/unit/applicants_test.rb:1:in `require'
/Users/Shilpa/CampusHunter/test/unit/applicants_test.rb:1:in `<top (required)>'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:10:in `require'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:10:in `block (2 levels) in <main>'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:9:in `each'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:9:in `block in <main>'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:4:in `select'
/Users/Shilpa/.rvm/gems/ruby-1.9.3-p194@global/gems/rake-0.9.2.2/lib/rake/rake_test_loader.rb:4:in `<main>'
